hpx/util/invoke.hpp

```diff
@@ -127,6 +127,23 @@ namespace hpx { namespace util
         };
     }
 
+    /// Invokes the given functional type f with the content of
+    /// the argument pack vs
+    ///
+    /// \param f A functional type providing an `operator()`
```

Comment:
I believe the term you are looking for is _callable object_ (`f` is not a type), which is comprised by _function objects_ and _member pointers_.